wiki/Tournaments/Official_support/en.md

```diff
@@ -93,7 +100,7 @@ In rare circumstances, some long-running community tournaments may qualify for i
 - The banner must clearly display the logo, motif AND name of your tournament, plus any information about the ongoing stage or section that is being advertised.
 - The banner must be cleanly designed with an emphasis on being mostly transparent and relatively unobtrusive visually.
 
-Please do not submit a main menu banner with your original request—you will be asked and pointed to this section if you are authorized to make use of one.
+ Please do not submit a main menu banner with your original request — you will be asked and pointed to this section if you are authorized to make use of one.
```

Comment:
Articles generally don't put spaces around em dashes. News posts have somewhat relaxed rules on writing style, which is why they do have spaces there by consensus, but no such thing has been decided for regular articles.

```suggestion
Please do not submit a main menu banner with your original request—you will be asked and pointed to this section if you are authorized to make use of one.
```